index.html

```diff
@@ -0,0 +1,199 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <link rel="stylesheet" href="style.css">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>my-portfolio</title>
+</head>
+<body>
+    <div class="container">
+        <header>
+            <nav>
+                <a href="#">
+                    <img class="my-logo" src="images/my-logo.png" alt="logo">
+                </a>
+                <a href="#">
+                    <img class="menu" src="images/union.png" alt="union">
+                </a>
+            </nav>
+        </header>
+        <section class="main-section">
+            <br>
+            <h1 class="main-text">I’m Godanah
+                Glad to see you!</h1>
+            <p class="content">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+            <br>
+            <p>LET'S CONNECT</p>
+            <br>
+            <div class="contact">
+                <ul class="list">
+                    <li><a href="#"><img src="images/vector-1.png" alt="vector1"></a></li>
+                    <li><a href="#"><img src="images/vector-2.png" alt="vector2"></a></li>
+                    <li><a href="#"><img src="images/vector-3.png" alt="vector3"></a></li>
+                    <li><a href="#"><img src="images/vector-4.png" alt="vector4"></a></li>
+                    <li><a href="#"><img src="images/vector-5.png" alt="vector5"></a></li>
+                </ul>
+            </div>
+        </section>
+    </div>
+
+    <div class="card">
+        <section class="profile-picture">
+            <img src="images/snapshot-profile-1.png" alt="Snapshot-profile">
+        </section>
+        <section class="tonic">
+            <h1>Tonic</h1>
+            <div>
+                <ul class="list">
+                    <li><p>CANOPY</p></li>
+                    <li><img src="images/counter-1.png" alt="counter1"></li>
+                    <li><p>Back End Dev</p></li>
+                    <li><img src="images/counter-2.png" alt="counter2"></li>
+                    <li><p>2015</p></li>
+                </ul>
+            </div>
+            <p class="contents">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <br>
+            <div class="buttons">
+                <ul class="list">
+                    <li><a href="#" class="button"><img src="images/html.png" alt=".html"></a></li>
+                    <li><a href="#" class="button"><img src="images/css.png" alt=".css"></a></li>
+                    <li><a href="#" class="button"><img src="images/javascript.png" alt=".js"></a></li>
+                </ul>
+                <a href="#" class="button"><img src="images/see-project.png" alt="Seeproject"></a>
```

Comment:
- [ ] I recommend you not to use an image of `See Project`. Instead, it would be better to use text instead.

- [ ] After changing this to a text, add the necessary styling required as indicated in the template. Active, Hover and Pressed. Do this for all your buttons.

Reply:
Thank you so much. I am proceeding to rectify the changes. Thank you for the guidance